packages/fakta-uttaksdokumentasjon/i18n/nb_NO.json

```diff
@@ -36,6 +36,9 @@
   "UttakDokumentasjonFaktaDetailForm.Header": "Detaljer",
   "UttakDokumentasjonFaktaDetailForm.Oppdater": "Oppdater",
   "UttakDokumentasjonFaktaDetailForm.Avbryt": "Avbryt",
+  "UttakDokumentasjonFaktaDetailForm.MorsStillingsprosent.Label": "Hvor mange prosent jobber mor?",
+  "UttakDokumentasjonFaktaDetailForm.MorsStillingsprosent.ReadMoreTittel": "Når mor jobber mindre enn 75%",
+  "UttakDokumentasjonFaktaDetailForm.MorsStillingsprosent.ReadMoreInnhold": "Når mor arbeider deltid, blir fars utbetaling av foreldrepenger redusert tilsvarende reduksjonen i hennes arbeidstid. Far får dermed lavere utbetaling, men hans periode blir ikke forlenget som ved gradering. Hvis mor for eksempel jobber 50 prosent, får far utbetalt 50 prosent av sin fulle dagsats. Hvis mor jobber 75 prosent av en full stilling eller mer, får likevel far utbetalt foreldrepenger etter sin fulle dagsats. Mer om <a>vilkår for fars uttak (åpnes i ny fane)</a>.",
```

Comment:
Litt forvirrende tekst? Et nytt forslag:

Når mor arbeider deltid under 75 prosent, blir fars utbetaling av foreldrepenger redusert tilsvarende mors stillingsprosent, og han får fulle trekkdager. Dersom han tar ut mindre foreldrepenger enn mors stillingsprosent vil han få utbetalt det han søker om, og antall trekkdager blir redusert i henhold til mors stillingsprosent (slik som ved gradering). Mer om <a>vilkår for fars uttak (åpnes i ny fane)</a>.